SubQuery Featured (#936)
Concordium/concordium.github.io: source/mainnet/net/indexers/subquery.rst

========
SubQuery
========

> SubQuery is a leading blockchain data indexer that provides developers with fast, flexible, universal, open source and decentralised APIs for web3 projects. SubQuery SDK allows developers to get rich indexed data and build intuitive and immersive decentralised applications in a faster and more efficient way. SubQuery supports 100+ ecosystems including Concordium, Ethereum, Polygon, Polkadot, Algorand, NEAR, and Avalanche.

Reviewer: ```suggestion
SubQuery is a leading blockchain data indexer that provides developers with fast, flexible, universal, open-source, and decentralized APIs for web3 projects. SubQuery SDK allows developers to get rich indexed data and build intuitive and immersive decentralized applications in a faster and more efficient way. SubQuery supports 100+ ecosystems including Concordium, Ethereum, Polygon, Polkadot, Algorand, NEAR, and Avalanche.
```

> Another one of SubQuery's competitive advantages is the ability to aggregate data not only within a chain but across multiple blockchains all within a single project. This allows the creation of feature-rich dashboard analytics and multi-chain block scanners.

Reviewer: ```suggestion
Another of SubQuery's competitive advantages is the ability to aggregate data not only within a chain but across multiple blockchains all within a single project. This allows the creation of feature-rich dashboard analytics and multi-chain block scanners.
```

> Other advantages include superior performance with multiple RPC endpoint configurations, multi-worker capabilities and a configurable caching architecture. To find out more, visit our [documentation](https://academy.subquery/).

Reviewer: ```suggestion
Other advantages include superior performance with multiple RPC endpoint configurations, multi-worker capabilities, and a configurable caching architecture. To find out more, visit SubQuery's `documentation <https://academy.subquery/>`_.
```

**Useful resources**:
- SubQuery Docs: [SubQuery Academy (Documentation)](https://academy.subquery.network/)
- Intro Quick Start Guide: [1. Create a New Project](https://academy.subquery.network/quickstart/quickstart.html)
- [Concordium Quick Start Guide](https://academy.subquery.network/quickstart/quickstart_chains/concordium.html)

Getting Started
---------------------

Take a look at this SubQuery Starter Project that introduces SubQuery's Concordium support by indexing [Concordium](https://github.com/subquery/concordium-subql-starter/tree/main/Concordium/concordium-testnet-starter).

> You can also follow along this [step by step guide](https://academy.subquery.network/quickstart/quickstart.html) to get familiar with SubQuery.

Reviewer: ```suggestion
You can also follow this [step-by-step guide](https://academy.subquery.network/quickstart/quickstart.html) to get familiar with SubQuery.
```

Running and Hosting your Concordium SubQuery APIs

> ---------------------

Reviewer: ```suggestion
-----------------------------------------------------------------------
```

SubQuery is open-source, meaning you have the freedom to run it in the following three ways:

> - Locally on your own computer (or a cloud provider of your choosing), [view the instructions on how to run SubQuery Locally](https://academy.subquery.network/run_publish/run.html).

Reviewer 2: Please use RST link syntax.
```suggestion
- Locally on your own computer (or a cloud provider of your choosing), `view the instructions on how to run SubQuery Locally <https://academy.subquery.network/run_publish/run.html>`_.
```

> - You can publish it to SubQuery's enterprise-level [Managed Service](https://managedservice.subquery.network/), where we'll host your SubQuery project in production ready services for mission critical data with zero-downtime blue/green deployments. There even is a generous free tier. [Find out how](https://academy.subquery.network/run_publish/publish.html).

Reviewer: ```suggestion
- You can publish it to SubQuery's enterprise-level [Managed Service](https://managedservice.subquery.network/), where we'll host your SubQuery project in production ready services for mission critical data with zero-downtime blue/green deployments. There is even a generous free tier. [Find out how](https://academy.subquery.network/run_publish/publish.html).
```

> - You can publish it to the decentralised [SubQuery Network](https://subquery.network/network), the most open, performant, reliable, and scalable data service for dApp developers. The SubQuery Network indexes and services data to the global community in an incentivised and verifiable way and supports Concordium from launch.

Reviewer: ```suggestion
- You can publish it to the decentralized [SubQuery Network](https://subquery.network/network), the most open, performant, reliable, and scalable data service for dApp developers. The SubQuery Network indexes and services data to the global community in an incentivized and verifiable way and supports Concordium from launch.
```
